app/services/import_rewards_csv_service.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class ImportRewardsCsvService
```

Comment:
Would be good to have few simple tests for service (one for "happy path and some for errors).